java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -372,6 +386,157 @@ public void computeResult_resultAlwaysInsideProvidedBoundaries() {
     }
   }
 
+  /** Calling computeConfidenceInterval before noised mean is computed throws an exception. */
+  @Test
+  public void computeConfidenceInterval_calledBeforeResult() {
+    assertThrows(IllegalStateException.class, () -> mean.computeConfidenceInterval(ALPHA));
+  }
+
+  @Test
+  public void computeConfidenceInterval_callsNoiseComputeConfidenceIntervalCorrectly() {
+    int maxPartitionsContributed = 1;
+    int maxContributionsPerPartition = 3;
+    double alpha = 0.5;
+    mean =
+        BoundedMean.builder()
+            .epsilon(EPSILON)
+            .delta(DELTA)
+            .noise(noise)
+            .maxPartitionsContributed(maxPartitionsContributed)
+            .maxContributionsPerPartition(maxContributionsPerPartition)
+            .lower(2.0)
+            .upper(10.0)
+            .build();
+
+    mean.addEntry(3.0);
+    mean.addEntry(7.0);
+    mean.computeResult();
+    mean.computeConfidenceInterval(alpha, alpha / 2);
+
+    // Confidence interval for normalized sum.
+    verify(noise)
+        .computeConfidenceInterval(
+            eq(/* x = x1 + x2 - midpoint * count = 3 + 7 - 6 * 2 = */ -2.0),
+            eq(/* l0Sensitivity = */ maxPartitionsContributed),
+            eq(/* lInfSensitivity = maxContributionsPerPartition * (upper - lower) / 2 = 3 * (10 - 2) / 2 = */ 12.0),
+            eq(EPSILON / 2.0),
+            eq(DELTA / 2.0),
+            eq(alpha / 2));
+
+    // Confidence interval for count.
+    verify(noise)
+        .computeConfidenceInterval(
+            eq(/* x = */ 2L),
+            eq(/* l0Sensitivity = */ maxPartitionsContributed),
+            eq(/* lInfSensitivity = maxContributionsPerPartition = */ (long) maxContributionsPerPartition),
+            eq(EPSILON / 2.0),
+            eq(DELTA / 2.0),
+            eq(/* alphaDen = (alpha - alphaNum) / (1 - alphaNum) = 0.25/0.75 = */ 1.0 / 3.0));
+  }
+
+  @Test
+  public void computeConfidenceInterval_positiveSumUpperBound() {
+    // Sum confidence interval.
+    mockDoubleConfInt(ConfidenceInterval.create(0, 5));
+    // Count confidence interval.
+    mockLongConfInt(ConfidenceInterval.create(2, 5));
+    mean.computeResult();
+
+    // mean_upperbound = sum_upperBound / count_lowerBound + midPoint =  5.0 / 2.0 + (1.0+9.0)/2.0 = 7.5
```

Comment:
can you make spaces in the statement consistent? and in other tests as well? (below)